src/prefect/server/utilities/http.py

```diff
@@ -0,0 +1,20 @@
+def should_redact_header(key: str) -> bool:
+    """Indicates whether an HTTP header is sensitive or noisy and should be redacted
+    from events and templates."""
+    key = key.lower()
+    if key in {"authorization", "traceparent", "via"}:
+        return True
+    if key.startswith("x-forwarded"):
+        return True
+    if key.startswith("x-b3"):
+        return True
+    if key.startswith("x-cloud-trace"):
+        return True
+    if key.startswith("x-envoy"):
+        return True
+    if key.startswith("x-nebula"):
+        return True
```

Comment:
is this pertinent for OSS?

Reply:
Ohh good point, this probably is not, will remove